downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -38,10 +38,46 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ## Runtime updates
 
-### New features
-
 ### Improvements
 
+#### Added shift, unshift, pop and remove operation support to tuples
+The runtime now supports the shift, unshift, pop and remove operations on tuples similar to how they work on arrays. The necessary checks have been implemented to make sure that the operations will work as expected without causing problems related to mutability and inherent type.
+
+A few examples regarding the use of the operations are given below.
+```ballerina
+[int, int...] tuple1 = [1, 2];
+int val1 = tuple1.shift();          // 1
+
+[string, string...] tuple2 = ["hello"];
+tuple2.unshift("world", "cat");     // ["hello", "world", "cat"]
+
+[int, string, int...] tuple3 = [1, "hello", 4];
+var val3 = tuple3.pop();            // 4
+
+[int,string, float...] tuple4 = [7, "hello", 67.5, 89.7];
+var val4 = tuple4.remove(2);        // 67.5
+```
+
+### New Runtime Java APIs
+
+#### A Runtime Java API to get whether the remote management is enabled
+
+A new runtime Java API is added to provide information whether the remote management is enabled through a build option.
+
+```java
+boolean isRemoteEnabled();
```

Comment:
Why did we go with isRemoteEnabled, rather than say isRemoteManagamentEnabled or isRemoteMgmtEnabled?

Reply:
I think @HindujaB should be able to better answer this.

Reply:
cc @warunalakshitha

Reply:
Considered options like, 
- `isRemoteEnabled`
- `isRemoteMgtEnabled` - we use this name in internal impelementations
- `isRemoteMgmtEnabled`
- `isRemoteManagamentEnabled`
But these seemed like longer names or there's inconsistency in finding the shorten form of `Management` - `mgt` or `mgmt`. 
@warunalakshitha WDYT?

Reply:
`isRemoteEnabled` doesn't really make sense, does it? Unless users are already familiar, they'll be confused - "remote what is enabled"? 

If `isRemoteManagamentEnabled` is too long, IMO, we can just pick one of `isRemoteMgtEnabled` or `isRemoteMgmtEnabled` and make sure we are consistent with it for whatever future additions/changes.

Reply:
We can go with  isRemoteManagamentEnabled even it is too long. Normally JAva aPIS use Management word instead of short names

Reply:
Some examples https://www.javadoc.io/static/javax.cache/cache-api/1.0.0/javax/cache/configuration/CompleteConfiguration.html#isManagementEnabled()

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/43266
@ravinperera00 we can modify the note accordingly.
```suggestion
boolean isRemoteManagementEnabled();
```

Reply:
Got it. I'll make the necessary changes.